shardingsphere-sql-parser/shardingsphere-sql-parser-postgresql/src/main/java/org/apache/shardingsphere/sql/parser/visitor/impl/PostgreSQLDALVisitor.java

```diff
@@ -33,26 +42,50 @@
  * DAL visitor for PostgreSQL.
  */
 public final class PostgreSQLDALVisitor extends PostgreSQLVisitor implements DALVisitor {
-    
+
     @Override
     public ASTNode visitShow(final ShowContext ctx) {
         return new ShowStatement();
     }
-    
+
     @Override
-    public ASTNode visitSet(final SetContext ctx) {
+    public ASTNode visitSetVariable(final SetVariableContext ctx) {
         SetStatement result = new SetStatement();
-        if (null != ctx.configurationParameterClause()) {
-            result.setVariable((VariableSegment) visit(ctx.configurationParameterClause()));
+        if (null != ctx.variableExpr()) {
+            List<VariableExpr> variableExprList = new ArrayList<VariableExpr>(1);
+            VariableExprContext variableExprContext = ctx.variableExpr();
+            VariableSegment variableSegment = null;
+            VariableValueSegment variableValueSegment = null;
+            String scopeType = variableExprContext.scope() == null ? null : variableExprContext.scope().getText();
```

Comment:
We prefer to use `null == xxx` to instead of `xxx == null`

Reply:
done